app/views/layouts/application.html.erb

```diff
@@ -46,7 +45,7 @@
         <!-- Add styling to center -->
         <p>Made with love by the #April2017 <a class="footer-link" href="http://makersacademy.com">Makers Academy</a> cohort</p>
       </div>
-
+      <%= javascript_include_tag 'application' %>
     </footer>
```

Comment:
Cannot find another method that works. Trying to identify a way in rails to call a unique js file instead of all is also a rabbit hole.